README.md

```diff
@@ -31,7 +31,10 @@ The main class is `org.opendatakit.briefcase.ui.MainBriefcaseWindow`. This repos
 - `org.opendatakit.briefcase.ui.MainClearBriefcasePreferencesWindow` clears Briefcase preferences
 - `org.opendatakit.briefcase.ui.MainFormUploaderWindow` uploads blank forms to Aggregate instances
  
+## Running the project
+ 
 To run the project, use the `run` Gradle task. To package a runnable jar, use the `jar` Gradle task.
+To try it, you can use the demo server. In the window that opens when running, choose connect then fill in the [demo sever URL](http://opendatakit.appspot.com) and use anonymous as username (no password required):
```

Comment:
Let's do [http://opendatakit.appspot.com](http://opendatakit.appspot.com) so it's easy to copy and paste. Also, you don't actually need to put anything in for username and password so you can say something like "leave username and password blank."